lib/engine.js

```diff
@@ -18,132 +18,139 @@
 'use strict';
 
 var path = require('path'),
-    dust = require('dustjs-linkedin'),
     utils = require('./utils'),
-    patch = require('./patch'),
-    River = require('./brook'),
+    Brook = require('./brook'),
     reqwire = require('./reqwire');
+var VError = require('verror');
 
+var isAbsolute = require('path-is-absolute');
 
-var loaders = {
-    dust: require('./reader/dust'),
-    js: require('./reader/js')
-};
-
+var fs = require('fs');
 
+var debug = require('debuglog')('adaro');
 
-function createRenderContext(options) {
-    var context;
+var freshy = require('freshy').freshy;
```

Comment:
mix of styles for variable declaration and line breaks. I like a declaration per line, between the two styles. not necessary to address for merge, just noting it in case you feel like tightening this up.